menu/menu.lua

```diff
@@ -38,45 +38,51 @@ menu.key_map = {
         state.debugOn = not state.debugOn
     end,
     p = function()
-        state.paused = not state.paused
-        menu.unload()
-        love.audio.play(sounds.chirp_down)
+        menu.load_map()
+
     end,
     up = function()
         cycleMaps.up()
     end,
     down = function()
         cycleMaps.down()
     end,
+    enter = function()
```

Comment:
Why not save the `if` statement above and make this `['return'] = function()`?

Reply:
Because I don't know Lua. I thought we have discussed this a few times. Changing though